ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> Only <ul> (and probably <ol>) are not wrapped in a <p>.

Yeah, non-clock elements are wrapped, block elements aren't (since they don't need it).

> This migration tries to mimic what would have been the resulting html text if the user were using tinyMCE but to write plain text.

I understood what you're trying to do.

I'm asking **why** you think it's necessary to do it. Because I don't think it is. :)